### DISCUSSION TITLE
I made some pre-commit hooks for fvm.

### DISCUSSION THREAD

**Discussion by motrieux-thomas** [leoafarias/fvm#580] [Show and tell]
Hi everyone 👋🏻 

Recently, I made some [pre-commit](https://pre-commit.com/) hooks related to fvm, I post that here if it could interest someone : https://github.com/motrieux-thomas/fvm-pre-commit

There is one hook to check that FVM is configured for the project, and two other to start 'flutter analyze / dart format' through fvm.

By the way, thank you for your awesome work with fvm 🎉

**Comment by leoafarias** [comment]
@motrieux-thomas very cool, will be adding to the website on the third party packages and resources
